translation: [-0.0271, 0.0247, -0.0035]
rotation: [0.0231, -0.7298, -0.6830, 0.0176]

body_frame: "vehicle"